src/events/commandUnknown.ts

```diff
@@ -30,7 +31,7 @@ export default class extends Event {
 	}
 
 	public runCommand(message: Message, command: Command) {
-		const commandHandler = cast<CommandHandler>(this.client.monitors.get('commandHandler'));
+		const commandHandler = cast<CommandHandler>(this.client.events.get('userMessageCommandHandler'));
```

Comment:
Enums.d.ts?

Reply:
I decided not to make it a constaant because two reasons:

- It's used only once.
- Sapphire will allow us to nuke the 99% of the code.